tests/InstrEngineTests/NaglerInstrumentationMethod/stdafx.h

```diff
@@ -46,4 +46,10 @@ using namespace ATL;
     do { if (FAILED(hr = (EXPR))) { ATLASSERT(FALSE); return hr; } } while (false)
 #endif
 
-#include "../../../src/Common.Headers/tstring.h"
\ No newline at end of file
+#include "../Common.Headers/tstring.h"
```

Comment:
>../ [](http://example.com/codeflow?start=10&length=3)

I don't think these parent directories are correct... Does it work because we're including the Common.Lib.Lib in the AdditionalDependencies?

Reply:
Header files impact the compiler, libs impact the linker. So they should be unrelated. Are you sure tstring is actually getting pulled in via the precompiled header? Most likely, this isn't being found and anything using tstring is getting the header pulled in some other way.

Reply:
These paths work since the project has an `AdditionalIncludeDirectories` on `$(InstrumentationEngineApiInc);` though it is confusing. I'll go ahead and update these paths to be relative to this project itself.